framework/sandstone.cpp

```diff
@@ -173,6 +173,7 @@ enum {
     use_predictable_file_names_option,
     version_option,
     weighted_testrun_option,
+    random_compatibility_mode,
```

Comment:
Sorting order. Repeats below for the `struct options` and for the `switch` case label.